solcjs

```diff
@@ -93,7 +94,13 @@ var output = JSON.parse(solc.compile(JSON.stringify({
     }
   },
   sources: sources
-})));
+}
+
+if (program.basePath) {
+  options.settings.basePath = basePath
```

Comment:
I don't think this is as easy as just forwarding the option. We have to modify the read callback.